octodns/provider/cloudflare.py

```diff
@@ -170,8 +190,8 @@ def zone_records(self, zone):
         return self._zone_records[zone.name]
 
     def populate(self, zone, target=False, lenient=False):
-        self.log.debug('populate: name=%s, target=%s, lenient=%s', zone.name,
-                       target, lenient)
+        self.log.debug('populate: name=%s, cdn=%s, target=%s, lenient=%s',
```

Comment:
Don't think it makes sense to have `cdn` in here, these generally just print the params directly to the function. Whether or not it's enabled show be provided by the `__init__` debug line.